### ISSUE TITLE
build fails using gitbook v3.0.2

### ISSUE THREAD

**Issue by diraulo** [GitbookIO/gitbook#1329]
I'm getting the following when trying to build using v3.0.2

```shell
Downloading source
Extracting source
GitBook version is 3.0.2 
Tweaking book.json
Installing plugins
info: installing 3 plugins using npm@3.9.2 
info:  
info: installing plugin "image-captions" 
info: install plugin "image-captions" (*) from NPM with version 0.4.0 
/tmp/book
└─┬ gitbook-plugin-image-captions@0.4.0 
  ├─┬ cheerio@0.20.0 
  │ ├─┬ css-select@1.2.0 
  │ │ ├── boolbase@1.0.0 
  │ │ ├── css-what@2.1.0 
  │ │ ├── domutils@1.5.1 
  │ │ └── nth-check@1.0.1 
  │ ├─┬ dom-serializer@0.1.0 
  │ │ └── domelementtype@1.1.3 
  │ ├── entities@1.1.1 
  │ ├─┬ htmlparser2@3.8.3 
  │ │ ├── domelementtype@1.3.0 
  │ │ ├── domhandler@2.3.0 
  │ │ ├── entities@1.0.0 
  │ │ └─┬ readable-stream@1.1.14 
  │ │   ├── core-util-is@1.0.2 
  │ │   ├── inherits@2.0.1 
  │ │   ├── isarray@0.0.1 
  │ │   └── string_decoder@0.10.31 
  │ ├─┬ jsdom@7.2.2 
  │ │ ├── abab@1.0.3 
  │ │ ├── acorn@2.7.0 
  │ │ ├── acorn-globals@1.0.9 
  │ │ ├── cssom@0.3.1 
  │ │ ├── cssstyle@0.2.36 
  │ │ ├─┬ escodegen@1.8.0 
  │ │ │ ├── esprima@2.7.2 
  │ │ │ ├── estraverse@1.9.3 
  │ │ │ ├── esutils@2.0.2 
  │ │ │ ├─┬ optionator@0.8.1 
  │ │ │ │ ├── deep-is@0.1.3 
  │ │ │ │ ├── fast-levenshtein@1.1.3 
  │ │ │ │ ├── levn@0.3.0 
  │ │ │ │ ├── prelude-ls@1.1.2 
  │ │ │ │ ├── type-check@0.3.2 
  │ │ │ │ └── wordwrap@1.0.0 
  │ │ │ └─┬ source-map@0.2.0 
  │ │ │   └── amdefine@1.0.0 
  │ │ ├── nwmatcher@1.3.7 
  │ │ ├── parse5@1.5.1 
  │ │ ├─┬ request@2.72.0 
  │ │ │ ├── aws-sign2@0.6.0 
  │ │ │ ├── aws4@1.4.1 
  │ │ │ ├─┬ bl@1.1.2 
  │ │ │ │ └─┬ readable-stream@2.0.6 
  │ │ │ │   ├── isarray@1.0.0 
  │ │ │ │   ├── process-nextick-args@1.0.7 
  │ │ │ │   └── util-deprecate@1.0.2 
  │ │ │ ├── caseless@0.11.0 
  │ │ │ ├─┬ combined-stream@1.0.5 
  │ │ │ │ └── delayed-stream@1.0.0 
  │ │ │ ├── extend@3.0.0 
  │ │ │ ├── forever-agent@0.6.1 
  │ │ │ ├─┬ form-data@1.0.0-rc4 
  │ │ │ │ └── async@1.5.2 
  │ │ │ ├─┬ har-validator@2.0.6 
  │ │ │ │ ├─┬ chalk@1.1.3 
  │ │ │ │ │ ├── ansi-styles@2.2.1 
  │ │ │ │ │ ├── escape-string-regexp@1.0.5 
  │ │ │ │ │ ├─┬ has-ansi@2.0.0 
  │ │ │ │ │ │ └── ansi-regex@2.0.0 
  │ │ │ │ │ ├── strip-ansi@3.0.1 
  │ │ │ │ │ └── supports-color@2.0.0 
  │ │ │ │ ├─┬ commander@2.9.0 
  │ │ │ │ │ └── graceful-readlink@1.0.1 
  │ │ │ │ ├─┬ is-my-json-valid@2.13.1 
  │ │ │ │ │ ├── generate-function@2.0.0 
  │ │ │ │ │ ├─┬ generate-object-property@1.2.0 
  │ │ │ │ │ │ └── is-property@1.0.2 
  │ │ │ │ │ ├── jsonpointer@2.0.0 
  │ │ │ │ │ └── xtend@4.0.1 
  │ │ │ │ └─┬ pinkie-promise@2.0.1 
  │ │ │ │   └── pinkie@2.0.4 
  │ │ │ ├─┬ hawk@3.1.3 
  │ │ │ │ ├── boom@2.10.1 
  │ │ │ │ ├── cryptiles@2.0.5 
  │ │ │ │ ├── hoek@2.16.3 
  │ │ │ │ └── sntp@1.0.9 
  │ │ │ ├─┬ http-signature@1.1.1 
  │ │ │ │ ├── assert-plus@0.2.0 
  │ │ │ │ ├─┬ jsprim@1.2.2 
  │ │ │ │ │ ├── extsprintf@1.0.2 
  │ │ │ │ │ ├── json-schema@0.2.2 
  │ │ │ │ │ └── verror@1.3.6 
  │ │ │ │ └─┬ sshpk@1.8.3 
  │ │ │ │   ├── asn1@0.2.3 
  │ │ │ │   ├── assert-plus@1.0.0 
  │ │ │ │   ├─┬ dashdash@1.13.1 
  │ │ │ │   │ └── assert-plus@1.0.0 
  │ │ │ │   ├── ecc-jsbn@0.1.1 
  │ │ │ │   ├─┬ getpass@0.1.6 
  │ │ │ │   │ └── assert-plus@1.0.0 
  │ │ │ │   ├── jodid25519@1.0.2 
  │ │ │ │   ├── jsbn@0.1.0 
  │ │ │ │   └── tweetnacl@0.13.3 
  │ │ │ ├── is-typedarray@1.0.0 
  │ │ │ ├── isstream@0.1.2 
  │ │ │ ├── json-stringify-safe@5.0.1 
  │ │ │ ├─┬ mime-types@2.1.11 
  │ │ │ │ └── mime-db@1.23.0 
  │ │ │ ├── node-uuid@1.4.7 
  │ │ │ ├── oauth-sign@0.8.2 
  │ │ │ ├── qs@6.1.0 
  │ │ │ ├── stringstream@0.0.5 
  │ │ │ └── tunnel-agent@0.4.3 
  │ │ ├── sax@1.2.1 
  │ │ ├── symbol-tree@3.1.4 
  │ │ ├── tough-cookie@2.2.2 
  │ │ ├── webidl-conversions@2.0.1 
  │ │ ├─┬ whatwg-url-compat@0.6.5 
  │ │ │ └── tr46@0.0.3 
  │ │ └── xml-name-validator@2.0.1 
  │ └── lodash@4.13.1 
  ├── q@1.4.1 
  └── semver@5.1.0 

info: >> plugin "image-captions" installed with success 
info:  
info: installing plugin "codeblock-filename" 
info: install plugin "codeblock-filename" (*) from NPM with version 0.0.7 
/tmp/book
├── gitbook-plugin-codeblock-filename@0.0.7 
└── gitbook-plugin-image-captions@0.4.0 

info: >> plugin "codeblock-filename" installed with success 
info:  
info: installing plugin "comment" 
info: install plugin "comment" (*) from NPM with version 1.0.0 
/tmp/book
├── gitbook-plugin-codeblock-filename@0.0.7 
├── gitbook-plugin-comment@1.0.0 
└── gitbook-plugin-image-captions@0.4.0 

info: >> plugin "comment" installed with success 
Building Site, will upload
info: 9 plugins are installed 
info: loading plugin "image-captions"... OK 
info: loading plugin "codeblock-filename"... OK 
info: loading plugin "comment"... OK 
info: loading plugin "highlight"... OK 
info: loading plugin "search"... OK 
info: loading plugin "lunr"... OK 
info: loading plugin "sharing"... OK 
info: loading plugin "fontsettings"... OK 
info: loading plugin "theme-default"... OK 
info: found 67 pages 
info: found 55 asset files 
warn: "config.options" property is deprecated, use "config.get(key)" instead 

TypeError: Invalid Version: *
```

Any idea what could be the issue? When i revert to v2.6.7 it builds just fine.

**Comment by SamyPesse** [comment]
This is an error with plugin `image-captions`, @todvora is working on a new version to support GitBook v3.

**Comment by todvora** [comment]
Hi,
Sorry for the delay. I just released ```0.4.1``` of ```image-captions```. The problem should be solved now. 

Best regards,
Tomas

**Comment by diraulo** [comment]
Thanks @SamyPesse and @todvora. I just ran a successful build using 3.0.2 😉
